src/Misc/layoutbin/actions.runner.plist.template

```diff
@@ -13,7 +13,9 @@
     <key>WorkingDirectory</key>
     <string>{{RunnerRoot}}</string>
     <key>RunAtLoad</key>
-    <true/>    
+    <true/>
+    <key>SessionCreate</key>
```

Comment:
not sure we can set this for everyone, it might break codesigning in some cases.

Reply:
Can I help alleviate that concern? I can list some example tests
The following behaves very differently if running the service via `./run.sh` or `./svc.sh` 
`- run: security list-keychains` 

My changes bring `./svc.sh` inline with `./run.sh`.

Would you recommend somehow conditionally generating the template instead?

Reply:
@TingluoHuang Can you pick up this PR again please?